ResearchKit/Common/ORKNavigableOrderedTask.h

```diff
@@ -98,6 +98,15 @@ ORK_CLASS_AVAILABLE
  */
 @property (nonatomic, copy, readonly) NSDictionary<NSString *, ORKStepNavigationRule *> *stepNavigationRules;
 
+/**
+ Determines whether this task should act as a linear ordered task when reporting progress or 
+ report no progress information.
```

Comment:
Maybe simplify a bit with something like: "Determines whether the task should report its progress as a linear ordered task or not. The default value of this property is \`NO\`".